doc/source/basics.rst

```diff
@@ -1,6 +1,6 @@
 .. currentmodule:: pandas
 
-.. ipython:: python
+.. ipython:: python 
```

Comment:
I don't understand why we're adding a white space after all directives. Is there a reason, or is it your editor that did those automatically?

Afaik this space is wrong, and they should be removed again.